app/src/shared/utils/hasFeatureFlag.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ *  here we define the flag names
+ */
```

Comment:
I feel like this is pretty self explanatory, the next lines and doesn't need a comment. Maybe just me. ;)